### PR TITLE
Add missing error message translation strings

### PR DESCRIPTION
**Pull request by nihonjinrxs**
Adding this as a draft PR for now. I'll add missing translation strings to this as we find them during testing over the next couple days. Once we feel things are looking good, we can set this for review and get it merged.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207564945811067

### CODE REVIEW

**Review by krisrjohnson21 — APPROVED** [review]
🧼

File: files/templates/mbta/login/messages/messages_es.properties [hunk 1]
```diff
@@ -3,8 +3,8 @@ doRegister=Crea tu cuenta MBTA
 noAccount=¿Nueva usuaria?
 doForgotPassword=¿Has olvidado tu contraseña?
 loginAccountTitle=Inicie sesión en su cuenta MBTA
-backToLoginRegistration=«Inicia sesión con tu cuenta actual
-backToLoginResetPassword=«Volver al inicio de sesión
+backToLoginRegistration=« Inicia sesión con tu cuenta actual
```
Comment (napilla) [inline files/templates/mbta/login/messages/messages_es.properties:6]: is this intended? (space between the quotes and beginning of line)

**Review by napilla — COMMENTED** [review]
I think it looks good overall, will leave the approval for someone with more familiarity of the project. Left one comment that appears to be an edit, not adding missing strings.

File: files/templates/mbta/login/messages/messages_es.properties (hunk 1 above)
Comment (nihonjinrxs) [reply files/templates/mbta/login/messages/messages_es.properties:6]: Yes, this is on a button and the `«` is used as an arrow to go back. The machine translation made those actual quote characters in most languages, so I had to fix that back.

Comment (napilla) [reply files/templates/mbta/login/messages/messages_es.properties:6]: ah got it

**Review by napilla — APPROVED** [review]

**Pull request merged**